CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.1.1]
+### Fixed
+- Move `@keystonehq/metamask-airgapped-keyring` to dependencies, fixing an accidental breaking change in [27.1.0] ([#757](https://github.com/MetaMask/controllers/pull/757))
+
 ## [27.1.0]
 ### Added
 - Now the `KeyringController` supports the `QRKeyring` from `@keystonehq/metamask-airgapped-keyring`. Developers can enable the import of accounts from a QR hardware wallet. A new optional parameter, `setAccountLabel` from the `PreferencesController`, should be passed to the `KeyringController` to enable this new functionality. ([#685](https://github.com/MetaMask/controllers/pull/685))
+  - **UPDATE:** This is an accidental breaking change. Consumers are encouraged to upgrade to [27.1.1].
```

Comment:
I don't think this is a breaking change :thinking: I mean, it is a change that is broken :smile: . But that's _usually_ not what "breaking change" means. We can just describe this as broken.

Also we should add ` [DEPRECATED]` after the release title. I can deprecate it on npm now as well.

Reply:
Okay :) 

I tried to add `[DEPRECATED]` after the release title. I think the lint script didn't like that, but I can try again.